pkgs/development/compilers/tvm/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit, cmake, git }:
+stdenv.mkDerivation rec {
+  pname = "tvm";
+  version = "0.7.0";
+  src = fetchgit {
+    url = "https://github.com/apache/incubator-tvm.git";
+    rev = "v${version}";
+    sha256 = "0qflpd3lw0jslyk5lqpv2v42lkqs8mkvnn6i3fdms32iskdfk6p5";
+  };
+  nativeBuildInputs = [ cmake git ];
```

Comment:
It looks like `git` is not needed.

Reply:
The build does have some logic for detecting if it is a git checkout, but the build can succeed without it with a warning I believe.

Reply:
Yes, it tries to get comit hash to embed it into binary (https://github.com/apache/incubator-tvm/pull/6280). But even `git` command is available, it will not work, since by default `fetchgit` removes `.git/` folder after cloning repo.

Reply:
Ahh I didn't know about the `.git/` bit, fixed!